lib/stack.js

```diff
@@ -1,19 +1,20 @@
 class Stack {
```

Comment:
👀 While your implementation does work and utilize Javascript arrays effectively, unfortunately the readme requested you implement stacks using the included LinkedList class.